app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+

```

Comment:
- [ ] I would recommend updating the code to match with wireframe design and styling. It's not signup but the design should be like this. Please update it.

### Given design

![Screenshot from 2021-12-01 03-39-07](https://user-images.githubusercontent.com/22434660/145859986-a7e77da8-6e30-45f1-a4c4-fbf0077779ed.png)

### Your design


![Screenshot from 2021-12-13 22-56-15](https://user-images.githubusercontent.com/22434660/145860088-a23f2de0-7be1-4684-816f-a5ab46b9e3d3.png)